acceptance/run.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+CONFIG_FILE=config
+SAMPLE_APP_DIR=sample-android-mixpanel-integration
+TESTS=(test.py)
+NODE_SERVER_PID=NULL
+SELEDROID_PID=NULL
+NPM_PACKAGES_REQUIRED=(express)
+
+VERBOSE=0
+
+function npm_package_check {
+	for package in $NPM_PACKAGES_REQUIRED
+	do
+		ls node_modules 2> /dev/null | grep $package > /dev/null
+		if [ $? != 0 ]
+		then
+			echo "$package is missing. installing.."
+			if [ $VERBOSE != 0 ]
+			then
+				sudo npm install $package --save
+			else
+				sudo npm install $package --save > /dev/null
+			fi
+		fi
+	done
+}
+
+function parse_config {
+	while IFS="=" read -r key value
+	do
+		case $key in
+			'MIXPANEL_API_TOKEN')
+				sed -i -e "s/MIXPANEL_API_TOKEN = \"YOUR API TOKEN\"/MIXPANEL_API_TOKEN = \"${value}\"/g" $SAMPLE_APP_DIR/src/com/mixpanel/example/hello/MainActivity.java
+			;;
+		esac
+	done < $CONFIG_FILE
+}
+
+function build_apk {
+	pushd ${SAMPLE_APP_DIR} > /dev/null
+	if [ $VERBOSE != 0 ]
```

Comment:
Let's not have switches- let's either always or never run it as verbose, and let users use grep if they need to see special stuff in the output